src/adapters/scene_category.js

```diff
@@ -4,14 +4,26 @@ import constants from '../../config/constants.yml';
 import Telemetry from 'src/libs/telemetry';
 import { toUrl } from 'src/libs/pois';
 import { fire, listen } from 'src/libs/customEvents';
-import { poisToGeoJSON, emptyFeatureCollection } from 'src/libs/geojson';
+import { poiToGeoJSON, emptyFeatureCollection } from 'src/libs/geojson';
 import { getFilteredPoisStyle } from 'src/adapters/pois_styles';
-import { createMapGLIcon, createIcon } from 'src/adapters/icon_manager';
 import { isMobileDevice } from 'src/libs/device';
+import { createMapGLIcon, createPinIcon } from 'src/adapters/icon_manager';
+import IconManager from 'src/adapters/icon_manager';
 
 const DYNAMIC_POIS_LAYER = 'poi-filtered';
 const mapStyleConfig = nconf.get().mapStyle;
 
+const poisToGeoJSON = pois => {
```

Comment:
I think we can move this function to `src/libs/geojson`, this way `poiToGeoJSON` and `poisToGeoJSON` are side by side.

Reply:
You're right in principle :) It was actually there at first, but referencing `IconManager` from `src/libs/geojson` made the unit tests crash.
The reason is `IconManager` itself imports a YAML file as as a dependency, but the yaml loader for Jest doesn't support the placeholder syntax we use in this file… This is a mess, I spent three whole hours on it, so to advance I moved the function calling IconManager in this file which isn't unit-tested.